src/pages/AddLiquidityV2/index.tsx

```diff
@@ -498,12 +502,18 @@ export default function AddLiquidity() {
               })
           })
           .catch((error: any) => {
-            console.error('Failed to send transaction', error)
             setAttemptingTxn(false)
-            // we only care if the error is something _other_ than the user rejected the tx
-            if (error?.code !== 4001) {
-              console.error(error)
-            }
+            // sending tx error, not tx execute error
+            const message = friendlyError(error)
+            console.error('Add liquidity error:', { message, error })
+            notify(
```

Comment:
will show when user reject?

Reply:
Yes. Currently we not showing anything, quite bad in UX